bootstrap/bootstrap.go

```diff
@@ -195,6 +195,11 @@ func Run(
 		handlers,
 	)
 
+	if !success {
+		cancel()
```

Comment:
can we log something here?

Reply:
This only occurs when a bootstrap handler fails. The handler will have logged an error message. I add comment why we don't need to log.